docs/management/connectors/action-types/slack.asciidoc

```diff
@@ -55,30 +67,47 @@ You can test connectors with the <<execute-connector-api,run connector API>> or
 as you're creating or editing the connector in {kib}. For example:
 
 [role="screenshot"]
-image::management/connectors/images/slack-params-test.png[Slack params test]
+image::management/connectors/images/slack-api-params.png[Slack API connector test]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
 Slack actions have the following properties:
 
-Message::   The message	text, converted to the `text` field in the Webhook JSON payload. Currently only the text field is supported. Markdown, images, and other advanced formatting are not yet supported.
+Channels::
+One or more channels that your Slack app has access to.
+This option is available only in the web API type of connector.
+
+Message:: The message	text, converted to the `text` field in the Webhook JSON payload.
```

Comment:
Is this description correct for the web API method?  
> Message:: The message	text, converted to the `text` field in the Webhook JSON payload. Currently only the text field is supported...

If not, perhaps something more general like this is sufficient: "The Slack message text, which cannot contain Markdown, images, or other advanced formatting."